extension/chrome/settings/inbox/inbox.ts

```diff
@@ -464,4 +481,16 @@ Catch.try(async () => {
       await renderInbox(labelId);
     }
   }
+
+  const acctEmails = await Store.acctEmailsGet();
+  const acctStorages = await Store.getAccounts(acctEmails, ['picture']);
+  for (const email of acctEmails) {
```

Comment:
Same thing with this functionality. Instead of copy-pasting it all, better move it to a common place, package it into a function and re-use. You could put most of the code plus the `menuAcctHtml` into a single appropriatelly named function in the `Settings` class, then re-use.